hack/release.sh

```diff
@@ -54,26 +54,13 @@ update_chart_yamls() {
     dir=(./helm ./deploy-all-in-one.yaml ./deploy-all-in-one-tls.yaml)
     for d in ${dir[@]}
     do
-        find $d -type f -exec sed -i "s/$previous_version/$version/g" {} \;
+        find $d -type f -name "*.yaml" -exec sed -i.bak "s/$previous_version/$version/g" {} \;
     done
 }
 
-publish_release() {
-    local version=$1
-
-    # create gh release
-    gothub release \
-	   --user infracloudio \
-	   --repo botkube \
-	   --tag $version \
-	   --name "$version" \
-	   --description "$version"
-}
-
```

Comment:
@pkosiec what version of goreleaser you have tested this with? I think for older releases we had to do it explicitly. I was wondering if we should have a pre-checks to make sure the tooling with correct versions are installed. This can be done separately.

Reply:
I tested that both with 1.1.0 and 1.8.3 (the latest Goreleaser).  This is the output from 1.8.3:

```
 • releasing...     
   • loading config file       file=.goreleaser.yml
   • loading environment variables
   • getting and validating git state
      • building...               commit=5ee6f56f9054e3289fef69145ca79c65639f4e19 latest tag=v0.13.0
   • parsing tag      
   • running before hooks
      • running                   hook=go mod download
   • setting defaults 
   • checking distribution directory
      • --rm-dist is set, cleaning it up
   • loading go mod information
   • build prerequisites
   • writing effective config file
      • writing                   config=dist/config.yaml
   • building binaries
      • building                  binary=dist/botkube_linux_amd64_v1/botkube
      • building                  binary=dist/botkube_linux_arm_7/botkube
      • building                  binary=dist/botkube_linux_arm64/botkube
   • archives         
      • creating                  archive=dist/botkube_0.13.0_linux_arm64.tar.gz
      • creating                  archive=dist/botkube_0.13.0_linux_amd64.tar.gz
      • creating                  archive=dist/botkube_0.13.0_linux_armv7.tar.gz
   • calculating checksums
   • docker images    
      • building docker image     image=ghcr.io/drill-bot/botkube:v0.13.0-amd64
      • building docker image     image=ghcr.io/drill-bot/botkube:v0.13.0-arm64
      • building docker image     image=ghcr.io/drill-bot/botkube:v0.13.0-armv7
   • storing release metadata
      • writing                   file=dist/artifacts.json
      • writing                   file=dist/metadata.json
   • publishing       
      • docker images    
         • pushing                   image=ghcr.io/drill-bot/botkube:v0.13.0-amd64
         • pushing                   image=ghcr.io/drill-bot/botkube:v0.13.0-arm64
         • pushing                   image=ghcr.io/drill-bot/botkube:v0.13.0-armv7
      • docker manifests 
         • creating                  images=[ghcr.io/drill-bot/botkube:v0.13.0-amd64 ghcr.io/drill-bot/botkube:v0.13.0-arm64 ghcr.io/drill-bot/botkube:v0.13.0-armv7] manifest=ghcr.io/drill-bot/botkube:v0.13.0
         • pushing                   manifest=ghcr.io/drill-bot/botkube:v0.13.0
      • scm releases     
         • creating or updating release repo=drill-bot/botkube tag=v0.13.0
         • uploading to release      file=dist/botkube_0.13.0_linux_arm64.tar.gz name=botkube_0.13.0_linux_arm64.tar.gz
         • uploading to release      file=dist/botkube_0.13.0_linux_amd64.tar.gz name=botkube_0.13.0_linux_amd64.tar.gz
         • uploading to release      file=dist/botkube_0.13.0_linux_armv7.tar.gz name=botkube_0.13.0_linux_armv7.tar.gz
         • uploading to release      file=dist/botkube_0.13.0_checksums.txt name=botkube_0.13.0_checksums.txt
   • announcing       
   • release succeeded after 92.62s
```

Here's the produced release for my sample bot account: https://github.com/drill-bot/botkube/releases/tag/v0.13.0

I totally agree - pre-check would be nice, but for now I put exact versions of required software inside the release instruction introduced in https://github.com/infracloudio/botkube-docs/pull/74/files 🙂 I think eventually it would be great to fully automate this and just run a GitHub Actions workflow with a given parameter - e.g.:

![Screenshot 2022-05-18 at 09 59 08](https://user-images.githubusercontent.com/7155799/168988197-5e251bd7-d306-41fe-9efb-3a8efa3ecb9a.png)

Cheers!

Reply:
Yeah, that sounds great. Do you mind creating an issue for that?

Reply:
@PrasadG193 here's the issue 🙂  https://github.com/infracloudio/botkube/issues/584

Reply:
Thank you :pray: